crates/rust-analyzer/Cargo.toml

```diff
@@ -89,4 +91,12 @@ mbe.workspace = true
 jemalloc = ["jemallocator", "profile/jemalloc"]
 force-always-assert = ["always-assert/force"]
 sysroot-abi = []
-in-rust-tree = ["sysroot-abi", "ide/in-rust-tree", "syntax/in-rust-tree"]
+in-rust-tree = [
+    "sysroot-abi",
+    "ide/in-rust-tree",
+    "syntax/in-rust-tree",
+    "parser/in-rust-tree",
+    "rustc-dependencies/in-rust-tree",
+    "hir-def/in-rust-tree",
+    "hir-ty/in-rust-tree",
+]
```

Comment:
Not a fan of this either, this will ultimately end up with mentioning almost all of our crates at some point

Reply:
Me too, maybe a non cargo cfg is a better fit here?

Reply:
Yes, I think it'd be better if we could make it so that this doesn't thread through all the cargo files if possible